submissions/BlessingOkoro/php/create.php

```diff
@@ -0,0 +1,61 @@
+<?php
+if (isset($_POST['submit'])) {
+    require "config.php";
+    require "helper.php";
+    require "fetch.php";
+
+    try {
+        $connection = new PDO($dsn, $username, $password, $options);
+
+        $newProduct = array(
+            "itemname" => $_POST['itemname'],
+            "quantity" => $_POST['quantity'],
+            "brand_id" => $_POST['brand'],
+        );
+
+        /**
+         * This is equivalent to
+         * INSERT INTO products (itemname, quantity) values (:itemname, :quantity)
+         */
+        $sql = sprintf(
+            "INSERT INTO %s (%s) values (%s)",
+            "products",
+            implode(", ", array_keys($newProduct)),
+            ":" . implode(", :", array_keys($newProduct))
+        );
+
+        $statement = $connection->prepare($sql);
+        $statement->execute($newProduct);
+    } catch (PDOException $error) {
+        echo $sql . "<br>" . $error->getMessage();
+    }
+}
+?>
+
+<?php if (isset($_POST['submit']) && $statement) { ?>
+    > <?php echo $_POST['itemname']; ?> successfully added.
+<?php } 
+?>
+
+<h2>Add A Product </h2>
+
+<form method="post">
+    <label for="itemname">Name of Product</label>
+    <input type="text" name="itemname" id="itemname">
+    <?php session_start(); ?>
```

Comment:
Hello @bnokoro . You do not need to use Sessions. This means that if the Sessions Array is not up to date or is non-existent, you would not have any brands. Kindly fetch the brands in this file like it is done in your overview.php file. Kindly refactor.